photon-core/src/main/java/org/photonvision/vision/processes/VisionModule.java

```diff
@@ -451,6 +451,10 @@ boolean setPipeline(int index) {
         } else {
             pipelineSettings.cameraRedGain = -1;
             pipelineSettings.cameraBlueGain = -1;
+
+            // All other cameras (than picams) should support AWB temp
```

Comment:
It looks like the ov9281 "supports" awb but it doesn't do anything. Do we want to do anything about that case? Just leave it?

Reply:
Meh. Yolo until someone complains.